demos/install/init.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+kubectl gadget undeploy || true
+kubectl delete ns demo || true
```

Comment:
What is this file supposed to do? Execute this commands **before** running the demo?

Reply:
Yes